src/frontend/src/pages/FinancePage/CreateReimbursementRequestForm/CreateReimbursementFormView.tsx

```diff
@@ -0,0 +1,230 @@
+                  render={({ field: { onChange, value } }) => (
+                    <Select
+                      onChange={(newValue) => onChange(newValue.target.value as ClubAccount)}
+                      value={value}
+                      error={!!errors.account}
+                    >
+                      {Object.values(ClubAccount).map((account) => (
+                        <MenuItem key={account} value={account}>
+                          {account}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                  )}
+                />
+              </FormControl>
+              <FormHelperText>{errors.account?.message}</FormHelperText>
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <FormLabel>Date of Expense</FormLabel>
+                <Controller
+                  name="dateOfExpense"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <DatePicker
+                      value={value}
+                      onChange={(newValue) => {
+                        onChange(newValue ?? new Date());
+                      }}
+                      renderInput={(params) => (
+                        <TextField {...params} error={!!errors.dateOfExpense} helperText={errors.dateOfExpense?.message} />
+                      )}
+                    />
+                  )}
+                />
+              </FormControl>
+            </Grid>
+            <Grid item container xs={6} spacing={2}>
+              <Grid item xs={12}>
+                <FormControl fullWidth>
+                  <FormLabel>Expense Type</FormLabel>
+                  <Controller
+                    name="expenseTypeId"
+                    control={control}
+                    render={({ field: { onChange, value } }) => (
+                      <Select
+                        onChange={(newValue) => onChange(newValue.target.value)}
+                        value={value}
+                        error={!!errors.expenseTypeId}
+                      >
+                        {allExpenseTypes.map((expenseType) => (
+                          <MenuItem key={expenseType.expenseTypeId} value={expenseType.expenseTypeId}>
+                            {expenseType.name}
+                          </MenuItem>
+                        ))}
+                      </Select>
+                    )}
+                  />
+                </FormControl>
+                <FormHelperText>{errors.expenseTypeId?.message}</FormHelperText>
+              </Grid>
+              <Grid item xs={12}>
+                <FormLabel>Total Cost</FormLabel>
+                <Typography variant="h6">${totalCost}</Typography>
+              </Grid>
+            </Grid>
+            <Grid item xs={6}>
+              <FormControl fullWidth>
+                <ReceiptFileInput />
+                <input
```

Comment:
why not put this in the ReceiptFileInput?

Reply:
it wasnt working, i cant remember exactly what the issue was but I remember trying and it didnt work...

Reply:
like the input itself wasnt registering adding files I think the onchange function was breaking or something

Reply:
word